styleguide/source/_patterns/01-molecules/10-search-results/00-search-result/search-result-external.twig

```diff
@@ -0,0 +1,13 @@
+<div class="search-result-external {{ class }}">
+   <a href="{{ url }}" class="search-result_primary-wrapper">
+     <div class="search-result_label">
+         {% include 'atoms-title-label' with { 'content' : 'AMA Wire', 'class': 'search-result' } %}
+         {% include 'atoms-arrow-external' %}
+     </div>
+     {% include 'atoms-h4' with { 'class' : 'search-result_title', 'content' : 'Visit AMA Wire to view news and more information related to this topic.' } %}
+   </a>
+     <div class="search-result_members-only-label-wrapper">
```

Comment:
There seems to be a discrepancy in the comps on this point. The columns comp clearly treats the members only link as distinct from the content link, and for consistency's sake, I stuck with that for the list pattern. We could do either one, but I'd like to keep them consistent.

![screen shot 2017-06-22 at 3 12 03 pm](https://user-images.githubusercontent.com/28711067/27453879-80560a90-575d-11e7-93ad-c6a03341e695.png)